### ISSUE TITLE
Add test scripts and CI

### ISSUE THREAD

**Issue by swinslow** [fossology/fossdriver#13]
Currently, the fossdriver repo does not include any test scripts or CI integration. Prior to an official release, it should include:

- scripts to enable spinning up a test Fossology server (presumably using the Fossology docker images)
- CI integration (perhaps Travis CI?)
- testing scripts to run through fossdriver functionality and ensure it works against the Fossology test server)

As a bonus, this could also be set up to test against multiple versions of Fossology, to help ensure that fossdriver stays current if/when the Fossology interface changes.

**Comment by sjha2048** [comment]
Hi @swinslow , I would like to work on this issue, can you please assign it to me?

**Comment by swinslow** [comment]
Hi @sjha2048, can you provide some thoughts on the approach you plan to take? 

Happy to assign it but just hoping to see some details about the plan that you have in mind, so that it doesn't deter other folks from working on it too. Thanks and I appreciate your offer to help!

**Comment by sjha2048** [comment]
Thanks for replying, for starting off I was planning to dockerize fossdriver and CI integration(I found it pretty similar to Atarashi) using Travis as fossology also uses Travis, perhaps we can split this issue into several issues for implementing the test scripts?

**Comment by sjha2048** [comment]
needed some more clarification on the testing part.

**Comment by swinslow** [comment]
Makes sense to split into separate issues for some of the precursor steps, in particular if you're planning to put together a Dockerfile for fossdriver. Feel free to file an issue and to start putting together PRs for any of this.

The test scripts will be the main part of this though. What sorts of tasks have you used fossdriver for, so far? How have you been using it in Fossology workflows?
